controls/radtilelist/styles-and-templates/styling-tilelist.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Styling TileList
+page_title: Styling TileList
+description: Styling TileList
+slug: tilelist-styling
+tags: styling
+published: True
+position: 0
+---
+
+# Styling TileList
+
+This topic will go through the process of styling the common elements of __RadTileList__.
+
+* [Styling the Tiles](#styling-the-tiles)
+* [Styling the TileGroupContainer](#styling-the-tilegroupcontainer)
+
+## Styling the Tiles
+
+For the purpose of this example, the following definition of __RadTileList__ will be used.
+
+#### __[XAML] Example 1: Defining RadTileList in XAML__
+{{region xaml-radtilelist-styling-0}}
+	<telerik:RadTileList >
+            <telerik:RadTileList.Groups>
+                <telerik:TileGroup Header="Europe" DisplayIndex="0">
+                    <telerik:TileGroup.Items>
+                        <telerik:Tile Content="Italy" />
+                        <telerik:Tile  Content="Germany" />
+                    </telerik:TileGroup.Items>
+                </telerik:TileGroup>
+                <telerik:TileGroup Header="Asia" DisplayIndex="1">
+                    <telerik:TileGroup.Items>
+                        <telerik:Tile Content="China" />
+                        <telerik:Tile Content="India" />
+                    </telerik:TileGroup.Items>
+                </telerik:TileGroup>
+            </telerik:RadTileList.Groups>
+        </telerik:RadTileList>
+{{endregion}}
+
+Then, defining a __Style__ for the __Tile__ can achieved as follows.
+
+#### __[XAML] Example 2: Defining Style for the Tiles__
+{{region xaml-radtilelist-styling-1}}

```

Comment:
Add explanation on how this Style is applied. Currently, if you are not very familiar with WPF you will wonder how the style is set to the controls and where it should be defined. You can add couple of sentences that tell the style is implicit and it will be applied globally to all controls of the specified TargetType in the Resources scope where is defined. For example: Application.Resources.